packages/back-end/src/services/stats.ts

```diff
@@ -67,6 +67,13 @@ export interface AnalysisSettingsForStatsEngine {
   max_dimensions: number;
 }
 
+export interface BanditSettingsForStatsEngine {
+  var_names: string[];
+  var_ids: string[];
```

Comment:
variations

Reply:
do we need both of these? any of these? the response object doesn't reference these variation ids at all.

Reply:
We use these in the query for constructing results going into the stats engine.  
https://github.com/growthbook/growthbook/blob/4b319072d263e77daef9bf9bd1eb17aba28cd359/packages/stats/gbstats/gbstats.py#L106